firmware/components/driver_framebuffer/include/driver_framebuffer_matrix.h

```diff
@@ -0,0 +1,112 @@
+#pragma once
+
+#include "sdkconfig.h"
+#include <stdio.h>
+#include <string.h>
+#include <stdint.h>
+#include "esp_system.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/*
+ * This file is NOT part of an external library but is rather created and maintained by the badge team.
+ * 
+ * DO NOT TOUCH THIS FILE UNLESS YOU UNDERSTAND MATRICES!
```

Comment:
I really don't like people screaming in text files. And the warning to not touch it is superfluous: people who know know linear algebra will get nightmares even looking at this code. So don't worry, they won't touch it.

Reply:
None of the framebuffer driver is part of an "external library" so I agree: better just remove this comment completely. Also: you can review any change someone wants to make to this file so no worries :-)